impl/client/src/main/javascript/web/pentaho/util/object.js

```diff
@@ -226,7 +226,8 @@ define(["./has"], function(has) {
      * @param {string} property - The name of property.
      * @param {Object} lcaExclude - A lowest-common-ancestor object whose inherited properties should
      * not be returned.
-     * @return {?Object} The [property descriptor]{@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty}.
+     * @return {?Object} The
+     * [property descriptor]{@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty}.
```

Comment:
We should not be adding the complete link in the documentation, the template already has logic in place to handle most links to mdn global objects reference documentation.

But this isn't one of them, so we need to add support for it before simplifying the link.